app/src/Containers/JokeForm/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useRef, useState } from "react"
+import { JokeInput, JokeWrapper, SubmitButton } from "./styles"
+import { useDispatch, useSelector } from "react-redux"
+import { addJoke } from "../../actions"
+
+const JokeForm = () => {
+ const jokeInput = useRef()
+ const dispatch = useDispatch()
+ const joke = useSelector(state => state.joke)
+ const [error,setError] = useState(joke.error)
+
+ const handleJokeCreation = () => {
+   const joke = jokeInput.current?.value
```

Comment:
use `currentTarget` b/c ```currentTarget is the element that the event was bound to. It never changes. In the sample code above, e.currentTarget is the element.
target is the element user clicked on, in the case of click event. It can be the original element or any of its children depending on where user clicks on exactly.```